Src/SpriteFont.cpp

```diff
@@ -479,9 +479,13 @@ XMVECTOR XM_CALLCONV SpriteFont::MeasureString(_In_z_ wchar_t const* text, bool
         UNREFERENCED_PARAMETER(advance);
 
         auto w = static_cast<float>(glyph->Subrect.right - glyph->Subrect.left);
-        auto h = static_cast<float>(glyph->Subrect.bottom - glyph->Subrect.top) + glyph->YOffset;
```

Comment:
Changing the existing behavior is a bit risky...

Can you instead go with:

```
auto h = static_cast<float>(glyph->Subrect.bottom - glyph->Subrect.top) + glyph->YOffset;

h = (ignoreWhitespace) ? pImpl->lineSpacing : std::max(h, pImpl->lineSpacing);
```

And make sure the test cases are updated?

> This seems more consistent with what you implemented for ``MeasureBounds``.
```

Reply:
yeah I can do that.

Reply:
done.  the unit tests required no change.  just to clarify, are you expecting the new behavior should only apply if !ignoreWhitespace as well?